Student_BCOs/Najuma_Babirye_BCO_000532.json

```diff
@@ -1,13 +1,13 @@
 {
     "bco_id": "http://biocomputeobject.org/BCO_000532",
-    "checksum": "e1236a35764f029bb3ba7b475fb79f192df26b6536fccecc9b319408c1ea50f4",
+    "checksum": "ee2a0e2898aaf9e2955d45a2caec8dc5ee20aa46187a49a533d41c9370b13c3f",
     "bco_spec_version": "1.3.0",
     "provenance_domain": {
         "name": "RNA sequencing and gene enrichment of Pancreatic CTCs",
         "version": "1.0.0",
         "license": "https://spdx.org/licenses/CC-BY-4.0.html",
         "created": "2019-10-08T21:19:52.713920",
-        "modified": "2019-10-09T02:50:38.318710",
+        "modified": "2019-10-16T20:46:21.197743",
         "contributors": [
```

Comment:
@nbabirye not sure why I didn't flag this before, but all the publication authors need to be contributors. Sorry about not assigning this issue to you earlier. If you fix this, I'll review again. #113

Reply:
@jpat1546,
I submitted a new commit with the changes for "authored by" added into the BCO. The BCO is ready for review again.
I just finished reassigning it to you.
thank you

Reply:
@nbabirye 2 more changes:

- In addition to "authoredBy", each publication author should also have the "derivedFrom" contribution type. Please add. See #113. 
- change review status of your BCO. See #148.

Once you make these changes in the BCO editor, please download, copy/paste JSON into this pull request. Thanks!

Reply:
@jpat1546 I'm still in Biochem lab,  (which started at 2pm) with limited access to a computer for this adjustment. If I can't get to it  before 5pm, I'll still commit the new change and let you know that I made it before submitting to precision FDA.